cli/src/main.rs

```diff
@@ -184,17 +184,17 @@ fn main() -> Result<(), Box<dyn error::Error>> {
         Arg::with_name("keypair")
             .short("k")
             .long("keypair")
-            .value_name("PATH")
+            .value_name("FILEPATH")
```

Comment:
For consistency, this should be `KEYPAIR`. This is just a signer input, and accepts the same types as the others you've changed to `KEYPAIR` (including "ASK" seed-phrase prompt). It probably should have an `is_valid_signer` validator on it, like those others.

Reply:
Yep, totally. I'll fix.